### DISCUSSION TITLE
Error loading page

### DISCUSSION THREAD

**Discussion by rwcloudridgeworks** [amidaware/tacticalrmm#1194] [General]
Error when loading pages 

This site can’t be reached

rmm.rwcloud.co.uk is currently unreachable.
ERR_CONNECTION_FAILED

server been rebooted and force update

**Comment by dinger1986** [comment]
certificate renewed? Services all running? Checked troubleshooting setups in docs.tacticalrmm.com?

**Comment by wh1te909** [comment]
run the server troubleshooting script and paste output here (redact sensitive info)

**Comment by rwcloudridgeworks** [comment]
Hi sorry what’s the command to renew the cert thought it did it automatically?

Steve

Kind regards
Stephen Oxley
Managing Director
01132166777Tel: 07793444814
***@***.***
2 Kerry Hill, ​Horsforth, Leeds. LS18 4AY
www.ridgeworks.co.uk
*Please note - Support requests must ​be 
​emailed ​to: ***@***.***
P Please consider the environment before printing this e-mail.
Ridgeworks DS Ltd accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. Any views or opinions presented in this email are solely those of the author and do not necessarily represent those of the company.Please click to view our terms and conditions or here for our Privacy Policy.
Ridgeworks DS LTD is a company registered in England and Wales under number -11011886  Vat :- GB336645094.
Registered office: 2 Kerry Hill Horsforth Leeds LS18 4AY
​

On 27 Jun 2022, at 21:04, dinger1986 ***@***.***> wrote:

﻿

certificate renewed? Services all running? Checked troubleshooting setups in docs.tacticalrmm.com?

**Reply by dinger1986** [reply]
https://docs.tacticalrmm.com/update_server/#keeping-your-lets-encrypt-certificate-up-to-date

**Comment by rwcloudridgeworks** [comment]
Please can u give me script

Kind regards
Stephen Oxley
Managing Director
01132166777Tel: 07793444814
***@***.***
2 Kerry Hill, ​Horsforth, Leeds. LS18 4AY
www.ridgeworks.co.uk
*Please note - Support requests must ​be 
​emailed ​to: ***@***.***
P Please consider the environment before printing this e-mail.
Ridgeworks DS Ltd accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. Any views or opinions presented in this email are solely those of the author and do not necessarily represent those of the company.Please click to view our terms and conditions or here for our Privacy Policy.
Ridgeworks DS LTD is a company registered in England and Wales under number -11011886  Vat :- GB336645094.
Registered office: 2 Kerry Hill Horsforth Leeds LS18 4AY
​

On 27 Jun 2022, at 21:10, Dan ***@***.***> wrote:
﻿

run the server troubleshooting script and paste output here (redact sensitive info)

**Reply by dinger1986** [reply]
everything is in the docs, please read them

**Comment by rwcloudridgeworks** [comment]
Hi see attached

[image0.png][image1.png]

Kind regards
Stephen Oxley
Managing Director
01132166777Tel: 07793444814
***@***.***
2 Kerry Hill, ​Horsforth, Leeds. LS18 4AY
www.ridgeworks.co.uk
*Please note - Support requests must ​be 
​emailed ​to: ***@***.***
P Please consider the environment before printing this e-mail.
Ridgeworks DS Ltd accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. Any views or opinions presented in this email are solely those of the author and do not necessarily represent those of the company.Please click to view our terms and conditions or here for our Privacy Policy.
Ridgeworks DS LTD is a company registered in England and Wales under number -11011886  Vat :- GB336645094.
Registered office: 2 Kerry Hill Horsforth Leeds LS18 4AY
​

On 27 Jun 2022, at 21:27, dinger1986 ***@***.***> wrote:

﻿

everything is in the docs, please read them

**Reply by dinger1986** [reply]
nothing attached, did you try and renew certs

**Comment by rwcloudridgeworks** [comment]
Hi see again got below errors  Daphne services not running
Nginx not running

Yes done the cert
[image0.png][image1.png]

Kind regards
Stephen Oxley
Managing Director
01132166777Tel: 07793444814
***@***.***
2 Kerry Hill, ​Horsforth, Leeds. LS18 4AY
www.ridgeworks.co.uk
*Please note - Support requests must ​be 
​emailed ​to: ***@***.***
P Please consider the environment before printing this e-mail.
Ridgeworks DS Ltd accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. Any views or opinions presented in this email are solely those of the author and do not necessarily represent those of the company.Please click to view our terms and conditions or here for our Privacy Policy.
Ridgeworks DS LTD is a company registered in England and Wales under number -11011886  Vat :- GB336645094.
Registered office: 2 Kerry Hill Horsforth Leeds LS18 4AY
​

On 27 Jun 2022, at 21:37, dinger1986 ***@***.***> wrote:

﻿

nothing attached, did you try and renew certs

**Reply by dinger1986** [reply]
after doing the cert did you run `update.sh --force`

**Comment by rwcloudridgeworks** [comment]
I’m getting the below


Starting nginx service
Job for nginx.service failed because the control process exited with error code.
See "systemctl status nginx.service" and "journalctl -xe" for details.

Kind regards
Stephen Oxley
Managing Director
01132166777Tel: 07793444814
***@***.***
2 Kerry Hill, ​Horsforth, Leeds. LS18 4AY
www.ridgeworks.co.uk
*Please note - Support requests must ​be 
​emailed ​to: ***@***.***
P Please consider the environment before printing this e-mail.
Ridgeworks DS Ltd accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. Any views or opinions presented in this email are solely those of the author and do not necessarily represent those of the company.Please click to view our terms and conditions or here for our Privacy Policy.
Ridgeworks DS LTD is a company registered in England and Wales under number -11011886  Vat :- GB336645094.
Registered office: 2 Kerry Hill Horsforth Leeds LS18 4AY
​

On 27 Jun 2022, at 21:42, dinger1986 ***@***.***> wrote:

﻿

after doing the cert did you run update.sh --force

**Reply by dinger1986** [reply]
Ok so did you run the commands journalctl and systemctl? See what the output is?

**Comment by rwcloudridgeworks** [comment]
Hi see below


***@***.***:~$ "systemctl status nginx.service"
systemctl status nginx.service: command not found
***@***.***:~$ systemctl status nginx.service
● nginx.service - A high performance web server and a reve>     Loaded: loaded (/lib/systemd/system/nginx.service; en>     Active: failed (Result: exit-code) since Mon 2022-06->       Docs: man:nginx(8)
    Process: 10107 ExecStartPre=/usr/sbin/nginx -t -q -g d>    Process: 10108 ExecStart=/usr/sbin/nginx -g daemon on;>lines 1-6/6 (END)


Kind regards
Stephen Oxley
Managing Director
01132166777Tel: 07793444814
***@***.***
2 Kerry Hill, ​Horsforth, Leeds. LS18 4AY
www.ridgeworks.co.uk
*Please note - Support requests must ​be 
​emailed ​to: ***@***.***
P Please consider the environment before printing this e-mail.
Ridgeworks DS Ltd accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. Any views or opinions presented in this email are solely those of the author and do not necessarily represent those of the company.Please click to view our terms and conditions or here for our Privacy Policy.
Ridgeworks DS LTD is a company registered in England and Wales under number -11011886  Vat :- GB336645094.
Registered office: 2 Kerry Hill Horsforth Leeds LS18 4AY
​

On 27 Jun 2022, at 22:13, dinger1986 ***@***.***> wrote:

﻿

Ok so did you run the commands journalctl and systemctl? See what the output is?

**Reply by dinger1986** [reply]
run `nginx -t` Have you tried any of the troubleshooting steps at all in the docs?

**Comment by rwcloudridgeworks** [comment]
Hi that’s running successfully 



Kind regards
Stephen Oxley
Managing Director
01132166777Tel: 07793444814
***@***.***
2 Kerry Hill, ​Horsforth, Leeds. LS18 4AY
www.ridgeworks.co.uk
*Please note - Support requests must ​be 
​emailed ​to: ***@***.***
P Please consider the environment before printing this e-mail.
Ridgeworks DS Ltd accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. Any views or opinions presented in this email are solely those of the author and do not necessarily represent those of the company.Please click to view our terms and conditions or here for our Privacy Policy.
Ridgeworks DS LTD is a company registered in England and Wales under number -11011886  Vat :- GB336645094.
Registered office: 2 Kerry Hill Horsforth Leeds LS18 4AY
​

**Reply by dinger1986** [reply]
I just checked and you are running apache on this server or doing something with proxys http://rmm.rwcloud.co.uk/ which is it?

You can request commercial support by emailing help@amidaware.com

**Comment by rwcloudridgeworks** [comment]
We should not be running either

So we need to remove them

Kind regards
Stephen Oxley
Managing Director
01132166777Tel: 07793444814
***@***.***
2 Kerry Hill, ​Horsforth, Leeds. LS18 4AY
www.ridgeworks.co.uk
*Please note - Support requests must ​be 
​emailed ​to: ***@***.***
P Please consider the environment before printing this e-mail.
Ridgeworks DS Ltd accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. Any views or opinions presented in this email are solely those of the author and do not necessarily represent those of the company.Please click to view our terms and conditions or here for our Privacy Policy.
Ridgeworks DS LTD is a company registered in England and Wales under number -11011886  Vat :- GB336645094.
Registered office: 2 Kerry Hill Horsforth Leeds LS18 4AY
​

On 27 Jun 2022, at 22:27, dinger1986 ***@***.***> wrote:

﻿

I just checked and you are running apache on this server or doing something with proxys http://rmm.rwcloud.co.uk/ which is it?

You can request commercial support by emailing ***@***.******@***.***>

**Reply by dinger1986** [reply]
I would suggest checking your server yes, someone must have changed something on it? One of your colleagues?

**Reply by wh1te909** [reply]
disable and remove apache, can't have both apache and nginx running at same time.

**Comment by rwcloudridgeworks** [comment]
Thanks Dan all sorted now

Kind regards
Stephen Oxley
Managing Director
01132166777Tel: 07793444814
***@***.***
2 Kerry Hill, ​Horsforth, Leeds. LS18 4AY
www.ridgeworks.co.uk
*Please note - Support requests must ​be 
​emailed ​to: ***@***.***
P Please consider the environment before printing this e-mail.
Ridgeworks DS Ltd accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. Any views or opinions presented in this email are solely those of the author and do not necessarily represent those of the company.Please click to view our terms and conditions or here for our Privacy Policy.
Ridgeworks DS LTD is a company registered in England and Wales under number -11011886  Vat :- GB336645094.
Registered office: 2 Kerry Hill Horsforth Leeds LS18 4AY
​

On 27 Jun 2022, at 22:39, Dan ***@***.***> wrote:

﻿

disable and remove apache, can't have both apache and nginx running at same time.

**Reply by wh1te909** [reply]
Thank @dinger1986 for figuring out you had apache running lol

**Comment by rwcloudridgeworks** [comment]
Good morning all

I have just tested rmm this morning and got the below error , I have done a restart and a force update

Steve

***@***.***


Kind regards
Stephen Oxley
Managing Director
01132166777Tel: 07793444814
***@***.***
2 Kerry Hill, ​Horsforth, Leeds. LS18 4AY
www.ridgeworks.co.uk
*Please note - Support requests must ​be 
​emailed ​to: ***@***.***
P Please consider the environment before printing this e-mail.
Ridgeworks DS Ltd accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. Any views or opinions presented in this email are solely those of the author and do not necessarily represent those of the company.Please click to view our terms and conditions or here for our Privacy Policy.
Ridgeworks DS LTD is a company registered in England and Wales under number -11011886  Vat :- GB336645094.
Registered office: 2 Kerry Hill Horsforth Leeds LS18 4AY
​

**Reply by dinger1986** [reply]
No screenshot.

Your certificate has maybe expired, follow the process in the docs to renew.

**Comment by rwcloudridgeworks** [comment]
Hi have also checked the cert and its active ,

The error is 0 when trying to login

Steve


Kind regards
Stephen Oxley
Managing Director
01132166777Tel: 07793444814
***@***.***
2 Kerry Hill, ​Horsforth, Leeds. LS18 4AY
www.ridgeworks.co.uk
*Please note - Support requests must ​be 
​emailed ​to: ***@***.***
P Please consider the environment before printing this e-mail.
Ridgeworks DS Ltd accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. Any views or opinions presented in this email are solely those of the author and do not necessarily represent those of the company.Please click to view our terms and conditions or here for our Privacy Policy.
Ridgeworks DS LTD is a company registered in England and Wales under number -11011886  Vat :- GB336645094.
Registered office: 2 Kerry Hill Horsforth Leeds LS18 4AY
​

**Reply by dinger1986** [reply]
that would be my suggestion anyway, follow the docs, also look for the error as thats written up as well
